### ISSUE TITLE
Implementation of Alcremie forms (the decoration)

### ISSUE THREAD

**Issue by PP-theSLAYER** [kwsch/PKHeX#2538]
While the form ID of Alcremie is implemented, the "sweets" decorated on it is not.
(form ID it is used for the "cream/swirl" it has)

I recommend using a dropdown box that only appears if the species is Alcremie.
The location of the byte that controls the sweet decorations on the .pk8 format is 0xE4

Below lists the "sweets" that decorate Alcremie. I tested these and confirm they're right

0x00 Strawberry
0x01 Berry
0x02 Love
0x03 Star
0x04 Clover
0x05 Flower
0x06 Ribbon

TL;DR: The current forms it has is handled fine by PKHeX.
The only thing missing is the decoration it has on it.

**Comment by kwsch** [comment]
https://github.com/kwsch/PKHeX/issues/2466
